tailwind.config.js

```diff
@@ -50,7 +50,12 @@ module.exports = {
       },
     },
     screens: {
-      xl: '1280px',
+      xxl: '1536px',
+      xl: '1440px',
+      l: '1024px',
+      md: '800px',
+      s: '600px',
+      xs: '414px',
```

Comment:
```suggestion
      xl: '1440px',
```

Keep only xl screen, all without xs all screens are mobile